src/components/Profile/components/SchedulePanel/index.jsx

```diff
@@ -15,18 +18,11 @@ import GordonLoader from 'components/Loader';
 import { formatDistanceToNow } from 'date-fns';
 import { Markup } from 'interweave';
 import { Component, Fragment } from 'react';
-import myschedule from 'services/myschedule';
 import schedulecontrol from 'services/schedulecontrol';
 import { gordonColors } from 'theme';
-import urlRegex from 'url-regex-safe';
 import EditDescriptionDialog from './components/EditDescriptionDialog';
-import RemoveScheduleDialog from './components/RemoveScheduleDialog';
 import GordonScheduleCalendar from './components/ScheduleCalendar';
-import MyScheduleDialog from './components/myScheduleDialog';
-
-// Default values
-const STARTHOUR = '08:00';
-const ENDHOUR = '22:00';
+import styles from './ScheduleHeader.module.css';
 
 const styles2 = {
```

Comment:
```suggestion
const styles2 = {
```

1) styles2 is not a clear/valid name
2) I'm not sure what `withStyles` does, but is there a reason it's just exporting `styles2`?